pyxem/generators/vdf_generator.py

```diff
@@ -71,11 +73,64 @@ def get_vdf_images(self,
         vdfs : Signal2D
             Signal containing virtual dark field images for all unique vectors.
         """
+        if self.vectors:
+            vdfs = []
+            for v in self.vectors.data:
+                disk = roi.CircleROI(cx=v[0], cy=v[1], r=radius, r_inner=0)
+                vdf = disk(self.signal,
+                           axes=self.signal.axes_manager.signal_axes)
+                vdfs.append(vdf.sum((2,3)).as_signal2D((0,1)).data)
+
+            vdfim = VDFImage(np.asarray(vdfs))
+
+            if normalize==True:
+                vdfim.map(normalize_vdf)
+
+        else:
+            raise ValueError("DiffractionVectors non-specified by user. Please "
+                             "initialize VDFGenerator with some vectors. ")
+
+        return vdfim
+
+    def get_concentric_vdf_images(self,
+                                  k_min,
+                                  k_max,
+                                  k_steps,
+                                  normalize=False):
+        """Obtain the intensity scattered at each navigation position in an
+        ElectronDiffraction Signal by summation in an
+        annlus of specified inner and outer radius.
+
+        Parameters
+        ----------
+        k_min : float
+            Minimum radius of the annular integration window in reciprocal
+            angstroms.
+
+        k_max : float
+            Maximum radius of the annular integration window in reciprocal
+            angstroms.
+
+        k_steps : int
+            Number of steps within the annular integration window
+
+        Returns
+        -------
+        vdfs : Signal2D
+            Signal containing virtual dark field images for all steps within
+            the annulus.
```

Comment:
This should be, as follows:

vdfs : VDFImage
VDFImage object containing ....

Reply:
The docstring for the other get_... function also needs this update by the way - this is from the initial implementation before the VDFImage class existed and is a documentation bug that we may as well fix here.